src/containerapp/azext_containerapp/commands.py

```diff
@@ -262,3 +262,9 @@ def load_command_table(self, args):
         g.custom_command('set', 'create_or_update_java_logger', supports_no_wait=True)
         g.custom_command('delete', 'delete_java_logger', supports_no_wait=True)
         g.custom_show_command('show', 'show_java_logger')
+
+    with self.command_group('containerapp env maintenance-config', is_preview=True) as g:
+        g.custom_command('add', 'add_maintenance_config')
+        g.custom_command('update', 'update_maintenance_config')
+        g.custom_command('remove', 'remove_maintenance_config')
```

Comment:
For remove we need to ask customer to confirm, which zunli also comment here : https://github.com/Azure/azure-cli-extensions/pull/8190#discussion_r1823730132

```suggestion
        g.custom_command('remove', 'remove_maintenance_config', confirmation=True)
```

Reply:
right, added confirmation